build/assets/topologyupdater/04_configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nfd-topology-updater-conf
+data:
+  nfd-topology-updater.conf: |
+    #excludeList:
+    ##  node1: [cpu]
+    ##  node2: [memory, example/deviceA]
+    ##  *: [hugepages-2Mi]
```

Comment:
Should we remove it? Uncomment?

Reply:
no, i am leaving it on purpose, as an example of what can be done/changed